🐙 destination-pgvector: run up-to-date pipeline [2025-05-17]

airbyteio opened the pull request:
# Update destination-pgvector

This PR was autogenerated by running `airbyte-ci connectors --name=destination-pgvector up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


**Note:** CATs were failing on some sort of diff in the SPEC for this connector. The same drifted spec is generated from the latest published version of this connector (created over 4 months ago). I assume someone forgot to update the test data rather than there was actual true drift in the connector that we don't expect / want. @aaronsteers wdyt?

## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- Update versions of libraries in poetry.: Successful

- PATCH bump destination-pgvector version to 0.1.3: Successful

- Build destination-pgvector docker image for platform(s) linux/amd64, linux/arm64: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=51728) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=51728) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=51728) it.</sub>

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
Update versions of libraries in poetry.
PATCH bump destination-pgvector version to 0.1.3
Create or update pull request on Airbyte repository
Add changelog entry

octavia-squidington-iii committed:
chore: auto-fix lint and format issues

dbgold17 committed:
Merge branch 'master' into up-to-date/destination-pgvector

dbgold17 commented:
/run-cat-tests

dbgold17 committed:
add functools as explicit dependency

dbgold17 committed:
update spec test data to align with what is returned

dbgold17 commented:
/run-cat-tests

dbgold17 commented:
CATs passing after explicitly requiring `jaraco.functools`

aaronsteers reviewed (approved):
NICE. 🚀
